config/express.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var app 		= require('express')();
+var bodyParser 	= require('body-parser');
+var config 		= require('../config');
+
+if(process.env.NODE_ENV == 'development') {
+	app.use(require('morgan')({ format: 'dev' }));
+}
+
+app.set('json spaces', 2);
+
+app.use(bodyParser.urlencoded({'extended': true }));
```

Comment:
Leave a space between braces and object contents. Also don't need to use quotes around object keys unless they contain some weird stuff...